chat/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -70,13 +70,15 @@ public PasswordEncoder passwordEncoder() {
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         http.cors(corsCustomizer -> corsCustomizer.configurationSource(request -> {
             CorsConfiguration config = new CorsConfiguration();
-            config.setAllowedOriginPatterns(List.of(allowedOrigins));
+            config.setAllowedOrigins(Collections.singletonList("http://localhost:4200"));
```

Comment:
Hardcode this values may be a bad idea, and I have already set allowed origins using properties in my commit.